src/Interpreters/RowRefs.cpp

```diff
@@ -162,6 +162,12 @@ class SortedLookupVector : public SortedLookupVectorBase
             std::lock_guard<std::mutex> l(lock);
             if (!sorted.load(std::memory_order_relaxed))
             {
```

Comment:
This seems unrelated to the changes. Have I broken something? 🤔

Reply:
No, just checked this file before.